docker-entrypoint-funcs.sh

```diff
@@ -570,10 +627,19 @@ _edbdocker_bootstrap_cb() {
         "    edgedb -u ${EDGEDB_SERVER_USER} --port=<published-port> \   "
         "        --password instance link --trust-tls-cert my_instance   "
```

Comment:
```suggestion
        "        instance link --trust-tls-cert my_instance   "
```

This `--password` is misleading if we're using `EDGEDB_SERVER_INSECURE_DEV_MODE`, and the CLI will anyway prompt for password even without `--password` if a password is needed.

Reply:
> and the CLI will anyway prompt for password even without `--password` if a password is needed.

Will it?  It didn't use to be the case.

Reply:
Yeah that's one of the different parts between `edgedb ...` and `edgedb instance link ...`, in order to support a fully-interactive `edgedb instance link` command without any parameters.

Reply:
Oh, the `link` subcommand does handle it differently.